icu4c/source/tools/icuexportdata/icuexportdata.cpp

```diff
@@ -27,6 +29,75 @@ UBool haveCopyright = TRUE;
 UCPTrieType trieType = UCPTRIE_TYPE_SMALL;
 const char* destdir = "";
 
+// Mask constants for modified values in the Script CodePointTrie, values are logically 12-bits.
+int16_t DATAEXPORT_SCRIPT_X_WITH_COMMON    = 0x0400;
+int16_t DATAEXPORT_SCRIPT_X_WITH_INHERITED = 0x0800;
+int16_t DATAEXPORT_SCRIPT_X_WITH_OTHER     = 0x0c00;
+
+// TODO(ICU-21821): Replace this with a call to a library function
```

Comment:
@echeran can you give me an idea of how this list was generated? We need to update it.

Reply:
It was very likely just creating a script to parse the first column of [`ScriptExtensions.txt` from UCD (probably from the state of data as of Unicode v14.0.0, judging from the date of the PR)](https://unicode.org/Public/14.0.0/ucd/ScriptExtensions.txt)